### ISSUE TITLE
Android 11 / Lineage 18.1 can choose to write external SD but not select it as folder

### ISSUE THREAD

**Issue by rk-zero** [ppareit/swiftp#168]
As the title says you can enable write on external storage for any folder (also external SD) but can only choose /storage/emulated/0 as a shared folder.

Simply enabling entering any text into the shared folder would do the trick, but it just opens a folder selector.

Thanks for the great program, really helpful!

**Comment by Xavron** [comment]
I'm expecting this to be fixed with my upcoming update if its taken in here or I may make a public repo for it to be built. There were various issues with that and especially with newer Android versions that I've mitigated.

**Comment by Wallby** [comment]
Still no update?

**Comment by Xavron** [comment]
> Still no update?

Yes, here on github. A short-ish time after my reply above ppareit added in the new code, and so sd card should work great on Android 11. I have further improvements pending as pull requests at this time.

However, swiftp on fdroid and Play Store are not updated at this time. They can be if/when ppareit gets around to it or if wanting to.

Although, Play Store is getting worse to deal with every year (its really bad now) so I don't blame anyone for not dealing with them. But, the app's pending updates are good to go for it :)
